project/Dependencies.scala

```diff
@@ -13,29 +13,29 @@ object Dependencies {
 
     val Scala210 = "2.10.7"
     val Scala212 = "2.12.8"
-    val Scala213 = "2.13.0-RC1"
+    val Scala213 = "2.13.0-RC2"
     val Scala    = Seq(Scala212, Scala213)
     val SbtScala = Seq(Scala212, Scala210)
 
     // If you update the version of Play, you probably need to update the other Play* variables.
-    val Play             = "2.7.3-SNAPSHOT"
-    val PlayJson         = "2.7.3"
-    val PlayStandaloneWs = "2.1.0-SNAPSHOT"
+    val Play             = "2.8.0-M1"
+    val PlayJson         = "2.8.0-M1"
+    val PlayStandaloneWs = "2.1.0-M2"
     val Twirl            = "1.4.1"
     val PlayFileWatch    = "1.1.8"
 
-    val Akka: String = sys.props.getOrElse("lagom.build.akka.version", "2.5.22")
-    val AkkaHttp     = "10.1.8+18-6d03ee45+20190508-0842"
+    val Akka: String = sys.props.getOrElse("lagom.build.akka.version", "2.5.23")
+    val AkkaHttp     = "10.1.8" // play 2.8.0-M1 is pulling 10.1.8+26-f33ec39a and it doesn't exist for 2.13.0-RC2
```

Comment:
It's in https://dl.bintray.com/akka/snapshots/com/typesafe/akka/akka-http-core_2.13.0-RC2/10.1.8+26-f33ec39a/.

Add `resolvers in ThisBuild += Resolver.bintrayRepo("akka", "snapshots")`